dev_guide/daemonsets.adoc

```diff
@@ -121,8 +121,10 @@ Node:        openshift-node02.hostname.com/10.14.20.137
 
 [IMPORTANT]
 ====
-Currently, updating a daemonset's pod template does not affect existing pod
```

Comment:
Maybe this line shouldn't have been removed, I haven't tested this.

Reply:
The existing document wasn't wrong, it was just incomplete in that if you change the nodeSelector in the pod template, that can cause pods to be assigned/removed from nodes with a potentially new template.  Nodes that still match the nodeSelector will not get the new pod template if it has been changed.

I would just add your clarifying sentence and not remove anything.

Reply:
@sjenning PTAL

Reply:
@mdshuai PTAL